django/template/engine.py

```diff
@@ -203,6 +203,13 @@ def render_to_string(self, template_name, context=None,
                          context_instance=_context_instance_undefined,
                          dirs=_dirs_undefined,
                          dictionary=_dictionary_undefined):
+        return ''.join(self.stream(template_name, context,
+                                   context_instance, dirs, dictionary))
+
+    def stream(self, template_name, context=None,
+               context_instance=_context_instance_undefined,
```

Comment:
Please create a `_stream()` helper with this signature and use it so that the deprecated parameters don't appear in the `stream()` signature.